source/MaterialXCore/Document.cpp

```diff
@@ -1054,7 +1054,10 @@ void Document::upgradeVersion()
         // If they are, rename the nodes.
         for (NodeGraphPtr nodegraph : getNodeGraphs())
         {
-            StringSet interfaceNames;
+            // Clear out any erroneously set version 
+            nodegraph->removeAttribute(Element::VERSION_ATTRIBUTE);
```

Comment:
As part of upgrade for now we strip out invalid cases. Can be removed post 1.38 release.

Reply:
Maybe just add a "TODO" code comment about removing this, if it can be removed later.